reactive/jenkins.py

```diff
@@ -355,3 +355,25 @@ def recover_jenkins(plugins):
     service_restart('jenkins')
     api.wait()  # Wait for the service to be fully up
     plugins.clean_backup()
+
+
+@when_not("config.default.update-center")
+@when("config.changed.update-center")
+def configure_update_center():
+    """ Change Update Center configuration when
+    config has changed.
```

Comment:
No need to break this onto a new line I think?

Reply:
Addressed in: e88dad6